.github/workflows/yaml-lint.yaml

```diff
@@ -1,8 +1,15 @@
 name: Yaml Lint
 on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
   pull_request:
-    paths:
-      - "**.yaml"
```

Comment:
this could be part of other commit(commit title `ci: always run all the actions `)?

Reply:
Done